apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -105,30 +102,30 @@ loop(Socket, Transport, State) ->
             ok = Transport:close(Socket)
```

Comment:
Remove the dynamic function call on line 102. Only modules that define callbacks should make dynamic calls.